node/src/Pos/Client/Txp/Util.hs

```diff
@@ -141,6 +145,10 @@ isCheckedTxError = \case
 -- Tx creation
 -----------------------------------------------------------------------------
 
+-- | Whether addresses can be spent entirely only.
+newtype UseGroupedInputs = UseGroupedInputs Bool
```

Comment:
This should be named something like an address (or input) selection policy, since that is what it is, and it should be a data type with one constructor per policy, including any policy params (so it is not an enumeration).